### PR TITLE
add documentation from provider to generated code

### PR DESCRIPTION
**Pull request by jlarfors**

### CODE REVIEW

File: pkg/internal/terrajen/generator.go [hunk 1]
```diff
@@ -213,6 +222,53 @@ type Schema struct {
...
+	}
+
+	docsURL := fmt.Sprintf(
+		"https://registry.terraform.io/providers/%s/%s/docs",
+		s.ProviderSource,
+		s.ProviderVersion,
+	)
+	str.WriteString("\n\nDocumentation: " + docsURL)
+
+	if s.Deprecated {
```
Comment (veggiemonk) [inline pkg/internal/terrajen/generator.go:265]: this should be first, no?

**Comment by veggiemonk** [comment]
This is where an `update` flag for the tests would be most useful.

**Comment by jlarfors** [comment]
> This is where an `update` flag for the tests would be most useful.

There is an update command, I just had not run it :)

**Comment by jlarfors** [comment]
> > This is where an `update` flag for the tests would be most useful.
> 
> There is an update command, I just had not run it :)

I spoke too soon... For the remaining failing tests there is no update. That took a bit of manual work :(

**Pull request merged**